tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -32,6 +32,14 @@ build_target="${build_target:-RELEASE}"
 
 ovmf_dir="${ovmf_repo##*/}"
 
+if [ "${ovmf_build}" == "tdx" ]; then
```

Comment:
Hi @Alex-Carter01 ,

I still don't get why it needs to overwrite `$ovmf_repo`  here. The caller of this script (i.e. tools/packaging/static-build/ovmf/build.sh) is going to get the repo, version, etc information to pass along the build-ovmf.sh script.

btw, by design the build scripit under tools/packaging/static-build/<component>/build-<component>.sh runs inside the container, which usually doesn't have `jq` and so `get_from_kata_deps` would fail. This the reason why the caller script  tools/packaging/static-build/<component>/build.sh is in charge of collecting all the need information to build the <component>.